instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/db/DbAttributesExtractor.java

```diff
@@ -25,7 +26,10 @@
   protected void onStart(AttributesBuilder attributes, REQUEST request) {
     set(attributes, SemanticAttributes.DB_SYSTEM, system(request));
     set(attributes, SemanticAttributes.DB_USER, user(request));
-    set(attributes, SemanticAttributes.DB_NAME, name(request));
+    AttributeKey<String> nameAttribute = dbNameAttribute();
```

Comment:
Hmm this weirdness seems like a good reason to go to the spec and remove the keyspace attribute? Not filling in a generic attribute seems like a bug, non-cassandra aware tracing systems would still get benefit from it.

Reply:
Definitely see where #2892 came from ;)

Reply:
👍 can I block my own PR 😂